src/main/java/Duke.java

```diff
@@ -1,10 +1,124 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+
+    public static ArrayList<Task> tasks = new ArrayList<Task>();
+    
+
+    public static void printLine(){
+        String horizontalLine = "\t――――――――――――――――――――――――――――――――――――――――――";
+        System.out.println(horizontalLine);
+    }
+    
+    public static void printWelcome(){
+        printLine();
+        System.out.println("\tWell hello there!\n" +
+                           "\tWhat can I do for you today?");
+        printLine();
+    }
+
+    public static void printGoodbye(){
+        printLine();
+        System.out.println( "\tThank you! See you again soon :)");
+        printLine();
+    }
+
+    public static void addEvent(String userInput){
+        int fromIndexStart = userInput.indexOf(" /from");
+        int fromIndexEnd = fromIndexStart + 7;
+        int toIndexStart = userInput.indexOf(" /to");
+        int toIndexEnd = toIndexStart + 5;
+        int startIndex = 5;
```

Comment:
Should you avoid the use of magic numbers and replace the number with a named constant instead?

Reply:
Try to avoid using constant, probably can use replace or substring command.